src/Servers/HttpSys/src/RequestProcessing/Response.cs

```diff
@@ -277,10 +278,25 @@ private void EnsureResponseStream()
     // What would we loose by bypassing HttpSendHttpResponse?
     //
     // TODO: Consider using the HTTP_SEND_RESPONSE_FLAG_BUFFER_DATA flag for most/all responses rather than just Opaque.
-    internal unsafe uint SendHeaders(HttpApiTypes.HTTP_DATA_CHUNK[]? dataChunks,
+    internal unsafe uint SendHeaders(ref UnmanagedBufferAllocator allocator,
+        Span<HttpApiTypes.HTTP_DATA_CHUNK> dataChunks,
         ResponseStreamAsyncResult? asyncResult,
         HttpApiTypes.HTTP_FLAGS flags,
         bool isOpaqueUpgrade)
+    {
+        fixed (HttpApiTypes.HTTP_DATA_CHUNK* chunks = dataChunks)
```

Comment:
Merge the internal and private SendHeaders together? Or is there a reason for them to be split (except the cosmetic reason of an extra indentation)?

Reply:
This was based on an earlier thought I had. It doesn't provide any value anymore.